### ISSUE TITLE
unused class StickerBean

### ISSUE THREAD

**Issue by kk63852** [fossasia/phimpme-android#2661]
**Describe the bug**

there are several used classes like Sticker Bean, Density Util inside editor package.

**Expected behavior**

unused class should be removed.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

**Comment by yashk2000** [comment]
@kk63852 I do not think this should be done. I did this once when I was removing all unused classes and variables, and it caused the build to fail. Some of the unused classes are used for the purpose of tests probably. Also, I am already working on it. I created an issue of removing unused methods. For these reasons, I request you to please close this issue.

**Comment by kk63852** [comment]
But these classes and their methods have no usages in any of the places. It's redundant to have these classes. I would suggest you navigate to these classes and look for their usages yourself and you'll believe it then.

**Comment by yashk2000** [comment]
I agree, but while I was working on #2032, I also tried removing all the unused classes. That created some errors in the build. I also know that these classes exist, but I did not open an issue to remove them again for this reason. 
That's the reason I opened an issue to remove just the unused methods present in classes which are being used to optimize the code. I'll again recommend you to not do. Also these classes might contain methods, that we could use to implement some new features in the future.
